features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesPresenter.kt

```diff
@@ -183,6 +182,18 @@ class MessagesPresenter @AssistedInject constructor(
             enableVoiceMessages = featureFlagsService.isFeatureEnabled(FeatureFlags.VoiceMessages)
         }
 
+        var dmUserVerificationState by remember { mutableStateOf<IdentityState?>(null) }
+
+        LifecycleResumeEffect(Unit) {
+            if (room.isDm && room.isEncrypted) {
```

Comment:
Maybe use `room.isDm` and `room.isEncrypted` as keys of `LifecycleResumeEffect`?